docs/content/mobile/android/cloud-deploy.md

```diff
@@ -25,11 +25,16 @@ Altenatively, if you want to create your own authentication server, follow [this
 To do this:
   *  From the [SpatialOS Console](https://console.improbable.io/projects), select your deployment name to display the project **OVERVIEW** screen.
   * In the **OVERVIEW** screen, there’s a **Tag** field, add `dev_login` to the field.
-1. [Create a Development Authentication Token (SpatialOS documentation).](https://docs.improbable.io/reference/latest/shared/auth/development-authentication#developmentauthenticationtoken-maintenance)
-1. Create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and includes the functionality you want. In your Unity Editor, add this script it to your Android client-worker GameObject.
-1. The `MobileWorkerConnector` provides a `DevelopmentAuthToken` field. Still in your Unity Editor, make sure your Android client-worker GameObject is selected and in the Inspector, locate the script you just added to it. 
-1. In the Inspector, in the script’s drop-down window, there is a field to add the authentication token that you created. 
-1. In the same drop-down window, ensure that the checkbox `ShouldConnectLocally` is not checked.
+1. [Create a Development Authentication Token (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/auth/development-authentication#developmentauthenticationtoken-maintenance).
+
+    > While developing your game you may want your game clients to use the proper authentication flow without already having developed your own authentication and login servers (following the process described [here](https://docs.improbable.io/reference/13.6/shared/auth/integrate-authentication-platform-sdk)). To facilitate this, Improbable hosts both a development authentication service and a development login service for use in early-stage game development.
+
+1. If your project does not already contain one, create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and include the functionality you want.
+1. In the Unity Editor, add the MonoBehaviour script to your Android client-worker GameObject. In the FPS Starter Project, navigate to **Assets** > **Fps** > **Prefabs** > **AndroidClientWorker**.
+1. The `MobileWorkerConnector` provides a `DevelopmentAuthToken` field. Still in the Unity Editor, make sure the Android client-worker GameObject is selected and in the Inspector, locate the script you just added to it.
```

Comment:
```suggestion
1. The `MobileWorkerConnector` provides a `DevelopmentAuthToken` field. Still in your Unity Editor, make sure the Android client-worker GameObject is selected and in the Inspector and locate the script you just added to it.
```
And makes the flow better. Small thing, not a biggee.

Reply:
Happy to go with this suggestion but wanted to suggest the following too: (The 2nd 'and' doesn't sound right when I'm reading it)
```1. The `MobileWorkerConnector` provides a `DevelopmentAuthToken` field. Still in your Unity Editor, make sure the Android client-worker GameObject is selected and locate the script you just added to it in the Inspector.```

Reply:
I've committed your suggestion.